ff3l-badblocks-disable-autoupdater/Makefile

```diff
@@ -0,0 +1,31 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ff3l-badblocks-disable-autoupdater
+PKG_VERSION:=1
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include ../../../package/gluon.mk
+
+define Package/ff3l-badblocks-disable-autoupdater
+  SECTION:=gluon
+  CATEGORY:=Gluon
+  TITLE:=Disable autoupdater if device has bad eraseblocks
+endef
+
+define Build/Prepare
+	mkdir -p $(PKG_BUILD_DIR)
+endef
+
+define Build/Configure
+endef
+
+define Build/Compile
+	$(call GluonSrcDiet,./luasrc,$(PKG_BUILD_DIR)/luadest/)
```

Comment:
Can we make sure that this package does nothing on later releases? Can we somehow check the current version and if >2019 just don't add any files?